attributes/default.rb

```diff
@@ -465,7 +465,7 @@
 
 #DOCKER
 default['hops']['docker']['enabled']                  = "true"
-default['hops']['docker_version']['ubuntu']           = "19.03.6-0ubuntu1~18.04.2"
+default['hops']['docker_version']['ubuntu']           = "19.03.6-0ubuntu1~18.04.3"
```

Comment:
Does it work if you replace the 3 with a `*`. Is not the first time this problem happens and it's not going to be the last.

Reply:
Yes, just tried it on a local vm and it seems to work